components/home/NavHome.tsx

```diff
@@ -15,11 +19,14 @@ import {
   MenuItem,
   MenuList,
   Text,
+  useBreakpointValue,
   useDisclosure,
+  VStack,
 } from '@chakra-ui/react';
 import type { Wallet as SolanaWallet } from '@solana/wallet-adapter-react';
 import { useWallet } from '@solana/wallet-adapter-react';
 import Avatar from 'boring-avatars';
+import Hamburger from 'hamburger-react';
```

Comment:
Not blocking for now, but generally don't use a package for 1 line functions. Make the app bloaty, which affects the performance.